pinot-core/src/test/java/com/linkedin/pinot/segments/v1/creator/BitmapInvertedIndexCreatorTest.java

```diff
@@ -78,7 +78,7 @@ public void testSingleValue() throws IOException {
     try (OnHeapBitmapInvertedIndexCreator onHeapCreator = new OnHeapBitmapInvertedIndexCreator(ON_HEAP_INDEX_DIR,
         COLUMN_NAME, CARDINALITY)) {
       for (int docId = 0; docId < NUM_DOCS; docId++) {
-        onHeapCreator.addSV(docId, dictIds[docId]);
```

Comment:
Can we add a test to cover the case where we add more than the earlier MAX values?